packages/context/src/binding.ts

```diff
@@ -112,6 +112,16 @@ export class Binding {
     return key;
   }
 
+  /**
+   * Concat the key and path to form a valid binding key
+   * @param key The key
+   * @param path the path
+   *
+   */
+  static concatKeyAndPath(key: string, path: string) {
```

Comment:
Maybe it's better to name the method as `buildKeyWithPath`.

Reply:
Can you refactor some tests to use this method?

Reply:
@truongminh Ping. Did you see my comments here?

Reply:
Sorry for my slow response.
I will refactor the code and write some tests for this feature tomorrow.